rabbitStairs/Program.cs

```diff
@@ -94,13 +69,39 @@ public int Rabbit(int k, int n)
             // Calculating the quantity of possibilities for current stairs
             int s = 0;
             // Trying all possibilities of jumping
-            for (int i = 1; i <= k; i++) 
+            for (var i = 1; i <= k; i++) 
             {
                 // Making the jump for i step will guide us to the same task with a stair that have (n - i) steps
-                s += Rabbit(k, n - i);
+                s += RabbitVariantCalculaiton(k, n - i);
             }
             return s;
         }
+
+        public void CheckInput(int k, int n)
+        {
+            if (k < 1)
+            {
+                Console.WriteLine("Quantity of jumps can't be less than 1. Program will be close.");
+                // Delay
+                Console.ReadLine();
+                Environment.Exit(-1);
```

Comment:
code duplication better to return something from this function and if input is wrong make
Console.ReadLine();
Environment.Exit(-1);